experiments/misc/ilamb_setup.jl

```diff
@@ -0,0 +1,146 @@
+import Dates
+
+"""
+    setup_run_for_ilamb(
+        ilamb_diagnostics_dir::String,
+        ilamb_models_parent_dir::String,
+        commit_id::String,
+    )
+
+Set up the ILAMB diagnostics produced from the run for ILAMB.
+
+This involves making the `MODELS` directory in `ilamb_models_parent_dir` if it
+does not exist, creating a model directory in the `MODELS` directory for this
+run, and creating symbolic links in the model directory pointing to NetCDF files
+in `ilamb_diagnostics_dir`.
+"""
+function setup_run_for_ilamb(
+    ilamb_diagnostics_dir::String,
+    ilamb_models_parent_dir::String,
+    commit_id::String,
+)
+    # Verify directories exist
+    isdir(ilamb_diagnostics_dir) ||
+        error("$ilamb_diagnostics_dir is not a directory")
+    isdir(ilamb_models_parent_dir) || error("$ilamb_root is not a directory")
+
+    # Make the MODELS directory
+    MODELS_dir = joinpath(ilamb_models_parent_dir, "MODELS")
+    if !isdir(MODELS_dir)
+        # This should not be called that often, because the MODELS directory
+        # should persist from run to run
+        mkdir(MODELS_dir)
+    end
```

Comment:
```suggestion
    # This should not be called that often, because the MODELS directory
    # should persist from run to run
    isdir(MODELS_dir) || mkdir(MODELS_dir)
```